instance_core_stats.tf

```diff
@@ -6,7 +6,7 @@ resource "openstack_compute_instance_v2" "grafana" {
   security_groups = ["default", "public-web2"]
 
   network {
-    name = "public"
+    name = "public-extended"
   }
 }
```

Comment:
Is it ok to switch the networks or do we need to keep the instance connected to both networks?